gitlab/v4/objects/projects.py

```diff
@@ -579,6 +579,29 @@ def mirror_pull(self, **kwargs: Any) -> None:
         path = f"/projects/{self.encoded_id}/mirror/pull"
         self.manager.gitlab.http_post(path, **kwargs)
 
+    @cli.register_custom_action("Project")
+    @exc.on_http_error(exc.GitlabCreateError)
+    def mirror_pull_details(self, **kwargs: Any) -> Dict[str, Any]:
+        """Get a project's pull mirror details.
+
+        Introduced in GitLab 15.5.
```

Comment:
Was this sentence pasted from the upstream docs or here intentionally? I'm fine with this, we probably just won't have it for most methods in the library though :)

Reply:
Yeah. I took it from the upstream docs.